docs/en/observability/cloud-monitoring/aws/monitor-aws-cloudtrail-firehose.asciidoc

```diff
@@ -0,0 +1,207 @@
++
+Open the log group where the CloudTrail service is sending the
+events. You must forward these events to an Elastic stack using the
+Amazon Data Firehose delivery stream. CloudWatch log group offers a
+https://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/Subscriptions.html[subscription filter] that allows you to choose log events from the log group and forward them to other services like Amazon Kinesis stream, an Amazon Data Firehose stream, or AWS Lambda.
+
+. Create a subscription filter for Amazon Data Firehose by following these steps.
+
+.. Choose the destination account.
++
+Select the delivery stream you created at step 3.
+
+.. Grant permission.
++
+Follow these steps to enable the CloudWatch service to send log events to the delivery stream in Amazon Data Firehose:
+
+... Create a new role with a trust policy that allows CloudWatch to assume the role.
++
+[source,json]
+----
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Principal": {
+                "Service": "logs.eu-north-1.amazonaws.com"
+            },
+            "Action": "sts:AssumeRole",
+            "Condition": {
+                "StringLike": {
+                    "aws:SourceArn": "arn:aws:logs:eu-north-1:<YOUR ACCOUNT ID>:*"
+                }
+            }
+        }
+    ]
+}
+----
+
+... Assign a new IAM policy to the role that permits ”putting records” into a
+in Amazon Data Firehose delivery stream.
++
+[source,json]
+----
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Action": "firehose:PutRecord",
+            "Resource": "arn:aws:firehose:eu-north-1:<YOUR ACCOUNT ID>:deliverystream/mbranca-dev-cloudtrail-logs"
+        }
+    ]
+}
+----
+
+When the new role is ready, you can select it in the subscription filter. Select *Amazon CloudTrail* in the log format option to configure log format and filters.
+
+[discrete]
+=== Verify
+
+To check if there are destination error logs, go to the AWS console, visit your Amazon Data Firehose delivery stream, and check for entries in the *Destination error logs*.
+
+If everything is correct, this list should be empty. If there’s an
+error, you can check the details. The following example shows a delivery stream that fails to send records to the Elastic stack due to bad authentication settings:
+
+image::firehose-failed-delivery-stream.png[Firehose failed delivery stream]
+
+The Amazon Data Firehose delivery stream reports the number of failed deliveries and failure details.
+
+[discrete]
+[[firehose-cloudtrail-step-five]]
+== Step 5: Visualize your CloudTrail logs in {kib}
+
+With the new subscription filter running, CloudWatch starts routing new
+CloudTrail log events to the Firehose delivery stream.
+
+image::firehose-monitor-cloudtrail-logs.png[Firehose monitor CloudTrail logs]
+
+Navigate to {kib} and choose among the following monitoring options:
```

Comment:
Is there a specific use case we could call out with each of these visualization/monitoring options?

Reply:
These are three different ways to visualize the same CloudTrail logs. But I'll defer to @zmoog to confirm.

Reply:
I considered one visualization to showcase a detailed view (Discover or Logs Explorer) and one high-level view (dashboard) of CloudTrail logs. 

As a user, I often take this two-level approach of getting the feel of the dataset from a quantitative perspective (how many events, etc) and a qualitative perspective (inspecting some sample documents).

Three ways are probably too many. 

@bmorelli25 @alaudazzi Is it better to offer one option only, or is there room for two?

Reply:
Promoting various visualization options is appropriate for tutorial-oriented content. We do the same in the [Monitor VPC Flow Logs](https://www.elastic.co/guide/en/observability/current/monitor-amazon-vpc-flow-logs.html) use case.